consumer/examples/bitbucket_10_api_example.e

```diff
@@ -27,7 +27,7 @@ feature {NONE} -- Initialization
 			service := api.with_api (create {OAUTH_10_BITBUCKET_API})
 								.with_api_key (api_key)
 								.with_api_secret (api_secret)
-								.with_callback ("http://www.eiffelroom.com")
+								.with_callback ("http://localhost:9090")
```

Comment:
Is it always port 9090 here?
what if a server is already listening on 9090?

Reply:
* It's the callback URL, so for testing, I'm using 9090, but a user can use a different url, like `www.example.com`.
* It's not a problem, it's part of the OAuth workflow, if we use it as a callback and we have the server running on it,  we will see something like http://localhost:9090?code=authorization code were we got the authorization code.